dmp_file.h

```diff
@@ -13,14 +11,14 @@ struct dmp_file_operator {
 };
 
 struct dmp_file {
-	uint8_t version, mode;
+	uint8_t version, mode, system;
 	uint8_t num_operators, lfo, fb, alg;
 	uint8_t lfo2;
 	struct dmp_file_operator operators[4];
 };
 
 void dmp_file_init(struct dmp_file *f);
-int dmp_file_load(struct dmp_file *f, uint8_t *data, size_t data_len, int system);
```

Comment:
Could you explain why you removed this field?

Reply:
As written above, the older versions of the DMP file format supported only YM2612 systems anyway, and for newer versions, the system information is part of the file itself, so it is not needed as a parameter.